.changeset/flat-laws-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"@eth-optimism/op-app": patch
+---
+
+Test
```

Comment:
I'm intentionally merging this in. Going to use this for publishing semver versions to NPM